src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultGetEventTriggerOperation.java

```diff
@@ -45,6 +45,16 @@ public DefaultGetEventTriggerOperation(@Qualifier(CachedCaseDefinitionRepository
         this.startEventOperation = startEventOperation;
     }
 
+    @Override
+    public CaseEventTrigger executeForCaseType(String caseTypeId, String eventTriggerId, Boolean ignoreWarning) {
+        return merge(startEventOperation.triggerStartForCaseType(caseTypeId,
```

Comment:
Maybe the return of `triggerStartForCaseType` could be capture in an intermediate variable to avoid the gaping space in the formatting and improve legibility.

Reply:
Done